inexor/engine/worldio.cpp

```diff
@@ -461,6 +461,7 @@ cube *loadchildren(stream *f, const ivec &co, int size, bool &failed);
 /// @param srcsurfs
 /// @param hassurfs
 /// @param normals
+/// @param hasnorms
```

Comment:
Your right, I was kind of confused about that one. If I remember correctly, this is one of the functions that I mentioned in the leftover document which mentioned that all the parameters weren't documented.